tests/ui/out_of_bounds_indexing/empty_array.stderr

```diff
@@ -0,0 +1,54 @@
+error: index out of bounds: the len is 0 but the index is 0
+  --> $DIR/empty_array.rs:6:5
+   |
+LL |     empty[0]; // Ok, let rustc's `const_err` lint handle `usize` indexing on arrays.
```

Comment:
I think emitting the lint here is fine, but could you remove the comment?

Reply:
Done :+1: